tracee-ebpf/tracee/pipeline.go

```diff
@@ -180,6 +180,17 @@ func (t *Tracee) prepareEventForPrint(done <-chan struct{}, in <-chan RawEvent)
 				StackAddresses, _ = t.getStackAddresses(rawEvent.Ctx.StackID)
 			}
 
+			// Currently, the timestamp received from the bpf code is of the monotonic clock.
+			// Todo: The monotonic clock doesn't take into account system sleep time.
+			// Starting from kernel 5.7, we can get the timestamp relative to the system boot time instead which is preferable.
+			if t.config.Output.MonotonicTS {
```

Comment:
do we really need to maintain this mode? feels like wall time is more useful for everyone